webpack.config.babel.js

```diff
@@ -7,170 +7,191 @@ import url from 'postcss-url';
-        unsafe: true,
-        unsafe_comps: true,
-        screw_ie8: true,
-        warnings: false
-      }
-    })
-  )
-}
-
-const configDist = {
+const baseConfig = {
   context: `${__dirname}/src`,
   entry: './index.js',
 
-  output: {
-    library: 'Onfido',
-    libraryTarget: 'umd',
-    path: `${__dirname}/dist`,
-    publicPath: '/',
-    filename: 'onfido.min.js'
-  },
-
   resolve: {
-    extensions: ['', '.jsx', '.js', '.json', '.less'],
-    modulesDirectories: [
-      `${__dirname}/src/lib`,
+    extensions: ['.jsx', '.js', '.json', '.less'],
+    modules: [
       `${__dirname}/node_modules`,
-      `${__dirname}/src`,
-      'node_modules'
+      `${__dirname}/src`
     ],
     alias: {
-      components: `${__dirname}/src/components`,    // used for tests
-      style: `${__dirname}/src/style`,
       'react': 'preact-compat',
       'react-dom': 'preact-compat',
       'react-modal': 'react-modal-onfido'
     }
   },
 
+  stats: { colors: true },
+
+  node: {
+    global: true,
+    process: false,
+    Buffer: false,
+    __filename: false,
+    __dirname: false,
+    setImmediate: false
+  },
+
+  devtool: ENV==='production' ? 'source-map' : 'eval-source-map'
+};
+
+
+const configDist = {
+  ...baseConfig,
+
+  output: {
+    library: 'Onfido',
+    libraryTarget: 'umd',
+    path: `${__dirname}/dist`,
+    publicPath: '/',
+    filename: 'onfido.min.js'
+  },
+
   module: {
-    loaders: [
-      ...baseLoaders,
+    rules: [
+      ...baseRules,
       {
         test: /\.(less|css)$/,
-        loader: ExtractTextPlugin.extract('style-loader', styleBaseLoaders.join('!'))
+        use: ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: baseStyleLoaders
+        })
       },
       {
         test: /\.(svg|woff2?|ttf|eot|jpe?g|png|gif)(\?.*)?$/i,
-        loader: 'file?name=images/[name]_[hash:base64:5].[ext]'
+        use: ['file-loader?name=images/[name]_[hash:base64:5].[ext]']
       },
       {
         test: /\.html$/,
-        loader: 'html-loader?interpolate'
+        use: ['html-loader?interpolate']
       }
     ]
   },
 
-  postcss: () => [
-    customMedia(),
-    autoprefixer({ browsers: 'last 2 versions' }),
-    url({
-      url: "inline"
-    })
-  ],
-
-  plugins: ([
+  plugins: [
     ...basePlugins,
-    new ExtractTextPlugin('style.css', {
+    new ExtractTextPlugin({
+      filename: 'style.css',
       allChunks: true,
       disable: ENV!=='production'
     }),
     new HtmlWebpackPlugin({
       template: './index.html',
       minify: { collapseWhitespace: true }
-    })
-  ]).concat(ENV==='production' ? [
-    new webpack.optimize.OccurenceOrderPlugin()
-  ] : []),
-
-  stats: { colors: true },
-
-  node: {
-    global: true,
-    process: false,
-    Buffer: false,
-    __filename: false,
-    __dirname: false,
-    setImmediate: false
-  },
-
-  devtool: ENV==='production' ? 'source-map' : 'eval-source-map',
+    }),
+    ... ENV === 'production' ?
```

Comment:
The webpack merge utility introduced in v2 is great for handling differences between different environments, so I'd recommend looking into adding it here :)